src/seedu/addressbook/ui/Formatter.java

```diff
@@ -54,6 +55,14 @@ private static String asIndexedList(List<String> listItems) {
         return formatted.toString();
     }
 
+    public static String getPrintableString(Printable... printables) {
+        StringBuilder sb = new StringBuilder();
+        for (Printable p : printables) {
+            sb.append(p.getPrintableString() + " ");
+        }
+        return sb.toString();
+    }
```

Comment:
Nice solution (although lacking header comments for the method :( )

A possible alternative to using `StringBuilder` is also Java's `StringJoiner`.